eng/pipelines/handlers.yml

```diff
@@ -279,7 +283,7 @@ stages:
             displayName: configure vsmac xcode
 
           - bash: |
-              dotnet restore Microsoft.Maui-net6.sln &&
+              dotnet build Microsoft.Maui.BuildTasks-net6.sln -c $(BuildConfiguration) -bl:$(LogDirectory)\$(BuildConfiguration)-buildtasks.binlog &&
```

Comment:
this doesn't work for the logs, i think is because of the path .. check how i did below.

Reply:
it writes ` -bl:/Users/runner/work/1/alogsDebug-buildtasks.binlog `

Reply:
thx